### ISSUE TITLE
setEditorState is not a function in defaultKeyCommands.js file

### ISSUE THREAD

**Issue by zhang-quan-yi** [draft-js-plugins/draft-js-plugins#1291]
How to happen: 
1. set custom `blockStyleFn` props;
2. set block type 

```javascript
RichUtils.toggleBlockType(
     this.state.editorState,
     blockType
)
```
3. press "Backspace" Key Util the editor content is empty

Here is the function call stack when "Backspace" key down:
file: `draft-js/lib/editOnKeyDown.js` 
![image](https://user-images.githubusercontent.com/12481444/61604433-1335fd80-ac74-11e9-8808-8e234b8acd3c.png)

file: `draft-js-plugins-editor/lib/Editor/defaultKeyCommands.js`
![image](https://user-images.githubusercontent.com/12481444/61604483-411b4200-ac74-11e9-978b-73de3cc91331.png)

**Comment by ynifamily3** [comment]
I triggered too.

**Comment by cmoad** [comment]
Make sure you are running draft-js 0.10.5 instead of 0.11.0. We hit this issue as well.

**Comment by nperichSYKES** [comment]
I don't know anything about draft or draft-plugins code, but maybe it's related to [this change in 0.11](https://github.com/facebook/draft-js/commit/0ecf9a63632db5612679dfc5934bf07e4c8cd8ae)?? handleKeyCommand in draft now accepts a third param. But draft-plugins was already using the third place for passing editorState...or something. I haven't yet worked out what's going on there, or why. 

Also, the draft-plugins [typescript types file](https://github.com/draft-js-plugins/draft-js-plugins/blob/93f9d32b703cf8e65e3576a5803388b362d19ba5/draft-js-plugins-editor/src/index.d.ts#L56) says that pluginFunctions is the fourth spot in handleKeyCommand, not the third, which the code with the error seems to use??

I'm not really sure if this is connected with the bug, or where to go with it. Just figured I'd drop these links with what little I've noticed so far in case anyone else finds it helpful.

**Comment by zhang-quan-yi** [comment]
my draft-js version is 0.11,0

**Comment by lvdang** [comment]
@zhang-quan-yi  - use the forth parameter 

Per DraftJS doc:
`handleKeyCommand?: (command: string, editorState: EditorState, eventTimeStamp: number) => DraftHandleValue`

e.g. 
 ` handleKeyCommand: (command, editorState, eventTimeStamp, { setEditorState }) `

**Comment by nperichSYKES** [comment]
Just to warn people even though this issue was closed, 0.11 will still break draft-js-plugins for at least the reason in this issue.

For example,
https://github.com/draft-js-plugins/draft-js-plugins/blob/1da9943359a7e3dd9076daef2b4bea9de0e34eae/draft-js-plugins-editor/src/Editor/defaultKeyCommands.js#L5

Also see,
https://github.com/draft-js-plugins/draft-js-plugins/issues/1297

**Comment by Flip437** [comment]
Upgrading to 0.11.0 at least, solved it for me.
Working with 0.11.7
